Makefile

```diff
@@ -212,7 +212,7 @@ mimir-build-image/$(UPTODATE): mimir-build-image/*
 # All the boiler plate for building golang follows:
 SUDO := $(shell docker info >/dev/null 2>&1 || echo "sudo -E")
 BUILD_IN_CONTAINER ?= true
-LATEST_BUILD_IMAGE_TAG ?= pr5955-facaa1c135
+LATEST_BUILD_IMAGE_TAG ?= pr5960-facaa1c135
```

Comment:
nit: we don't need this change, #5960 was cherrypick of #5955, but bot decided to rebuild the image.

Reply:
but don't bother fixing it, the two build images are the same.

Reply:
Thanks. I saw the changelog with two PRs and just took the latest.  I'll leave this as is.